projects/addon-preview/README.md

```diff
@@ -17,7 +17,7 @@ of multiple base libraries and several add-ons
 Install base packages:
 
 ```
-npm i @taiga-ui/{cdk,core,kit,icons}
+npm i @taiga-ui/{cdk,core,kit,icons,styles}
```

Comment:
It's not a base package, instead mention this package where there's a section about global styles

Reply:
fixed